services/mirror/mirror.go

```diff
@@ -290,97 +292,103 @@ func Update() {
 func SyncMirrors() {
 	// Start listening on new sync requests.
 	for repoID := range mirrorQueue.Queue() {
-		log.Trace("SyncMirrors [repo_id: %v]", repoID)
-		mirrorQueue.Remove(repoID)
+		syncMirror(repoID)
```

Comment:
I prefer to send another PR to improve this code.

Reply:
It's not a real refactor - extracting the function simply makes closing the git repo easier - otherwise you can't just use defer.